graphics/managed_surface.cpp

```diff
@@ -314,7 +316,7 @@ void ManagedSurface::blitFromInner(const Surface &src, const Common::Rect &srcRe
 
 		// For paletted format, assume the palette is the same and there is no transparency.
 		// We can thus do a straight copy of the pixels.
-		if (destFormat.bytesPerPixel == 1 && noScale) {
+		if (destFormat == Graphics::PixelFormat::createFormatCLUT8() && noScale) {
```

Comment:
You have a lot of destFormat == Graphics::PixelFormat::createFormatCLUT8(). It's unclear if this comparison compares palettes. Can we have a method .isCLUT8()

Reply:
I like this idea, instead of endless creation of the PixelFormat object

Reply:
@ccawley2011 I implemented this method. Please us it like `if (destFormat.isCLUT8())`

Reply:
OK, done.